extensions/resteasy-reactive/rest/spi-deployment/src/main/java/io/quarkus/resteasy/reactive/server/spi/AllowNotRestParametersBuildItem.java

```diff
@@ -0,0 +1,6 @@
+package io.quarkus.resteasy.reactive.server.spi;
+
+import io.quarkus.builder.item.SimpleBuildItem;
+
+public final class AllowNotRestParametersBuildItem extends SimpleBuildItem {
```

Comment:
A little Javadoc on what this done would be great

Reply:
Done